filter_plugins/ec2_custom.py

```diff
@@ -23,7 +23,7 @@ def validate_ruleset(ports, proto):
         raise errors.AnsibleFilterError('ports is empty or missing')
 
 
-def rules_from_dict(rules, src_list=None, use_nat_gw=False, nat_gw_src=None):
+def rules_from_dict(rules, src_list=None, sg_names_list=None, use_nat_gw=False, nat_gw_src=None):
```

Comment:
Parameters are documented, but not purpose of the function or any examples.
Any tests around this function?